### PR TITLE
Fix tests on newer versions of alcotest

### PR DESCRIPTION
**Pull request by psafont**
A testcase suite was repeated which causes newer alcotest version to fail.

The crowbar tests got simplified as well.

### CODE REVIEW

File: lib_test/crowbar_tests.ml [hunk 1]
```diff
@@ -76,34 +73,24 @@ let ds_value =
...
-      Cb.guard (min < max); ds_create "derive" ~min ~max Derive v);
-    Cb.map Cb.[ds_value; float; float] (fun v min max ->
-      let min, max = castd2s min max in
-      Cb.guard (min < max); ds_create "absolute" ~min ~max Absolute v);
-    Cb.map Cb.[ds_value; float; float] (fun v min max ->
-      let min, max = castd2s min max in
-      Cb.guard (min < max); ds_create "gauge" ~min ~max Gauge v)
-  ]
+  let ds_type = Cb.(choose [const Derive; const Absolute; const Gauge]) in
+  Cb.(map [ds_value; float; float; ds_type] (fun v x y typ ->
```
Comment (edwintorok) [inline lib_test/crowbar_tests.ml:77]: I was wondering why this compiles, looks like they overrode the syntax for list inside the Crowbar module.

**Review by edwintorok — APPROVED** [review]
I think you're missing a signed-off-by, see DCO bot.

File: lib_test/crowbar_tests.ml [hunk 2]
```diff
@@ -2,42 +2,39 @@ module Fring = Rrd.Fring
...
 module Cb = Crowbar
 
-(* cast double-precision float to single-precision *)
+(* cast double-precision floats to single-precision and order them *)
 let castd2s min max =
   let _data = Bigarray.Array1.create Bigarray.float32 Bigarray.c_layout 1 in
   Bigarray.Array1.set _data 0 min;
   let min = Bigarray.Array1.get _data 0 in
   Bigarray.Array1.set _data 0 max;
   let max = Bigarray.Array1.get _data 0 in
```
Comment (lindig) [inline lib_test/crowbar_tests.ml:11]: The `min`, `max` as name are not too helpful if indeed they might be reversed.

**Review by lindig — APPROVED** [review]

**Comment by psafont** [comment]
These changes do not need a release / tag

**Pull request merged**